lib/dpul_collections_web/set_locale_hook.ex

```diff
@@ -0,0 +1,13 @@
+defmodule DpulCollectionsWeb.SetLocaleHook do
```

Comment:
Could we add a module doc that says something like, "set up locale for front-end use"